### PR TITLE
chore: Cleanup Microsoft links

### PR DESCRIPTION
**Pull request by nschonni**
Using the MS Docs Author Pack "Cleanup Microsoft Links" function.
- Change MS Links to HTTPS
- Remove link locals

### CODE REVIEW

**Comment by PRMerger20** [comment]
@nschonni : Thanks for your contribution! The author, @mikeblome, has been notified to review your proposed change.

**Comment by nschonni** [comment]
This is an automated cleanup using https://github.com/Microsoft/vscode-docs-authoring/tree/develop/docs-markdown#cleanup-scripts
Since it touches so many files I can break it into smaller chunks if preferred, or you can run it internally yourselves

**Comment by ktoliver** [comment]
@gewarren - Could you review the proposed update? Thanks.

**Comment by gewarren** [comment]
We should run this internally so we can verify the links still work with locales removed (in some cases, they don't).

**Comment by nschonni** [comment]
FYI, this is the regexes that they use to update the links. I think it should all be safe, since they don't include stuff like the "forums" URLs that do break without the locales https://github.com/microsoft/vscode-docs-authoring/blob/dd51c6de8ce1a86b78b128ca301aaaab4a79b672/docs-markdown/src/controllers/cleanup-controller.ts#L535-L553

**Comment by gewarren** [comment]
@thomps23 Can you see about queuing up this work for the vendors?

**Comment by mikeblome** [comment]
@gewarren @nschonni This seems fine as an intermediate step. Ultimately all URLs going to docs.microsoft.com should be made into site-relative format, and all msdn.* URLs should be converted to some appropriate site-relative docs.microsoft.com URL or else deleted.

**Comment by Jak-MS** [comment]
Hi @gewarren @mikeblome - is there any action for this PR at this time?

Let us know if want any of these:

- `#sign-off` if it's ready to merge
- `#hold-off` if it requires changes before it's ready for sign off
- `#please-close` - if these changes aren't going to be merged

Thanks!

**Comment by nschonni** [comment]
@gewarren I can do either the PR split or commit split in this PR if you want to pick a file count for the split (instead of pulling this internally)

**Comment by gewarren** [comment]
@nschonni I'd say 100 files/commit. Is there any easy way to do that? I usually just do a commit per subfolder of docs if I'm making a large change across the repo.

**Comment by PRMerger12** [comment]
@nschonni : Thanks for your contribution! The author, @mikeblome, has been notified to review your proposed change.

**Comment by nschonni** [comment]
@gewarren tried to do the commit per folder (and on for each subfolder of vs-2015)

File: docs/cross-platform/using-visual-studio-tools-for-unity.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ Note that the scenario described here assumes that you have the source code—th
...
    
-   More information about PDB generation can be found [here](https://docs.microsoft.com/en-us/visualstudio/debugger/how-to-set-debug-and-release-configurations?view=vs-2019). If you are targeting the new runtime, please make sure that "Debugging Information" is set to "Portable", in order to properly generate Portable-PDB. If you are targeting the legacy runtime, you need to use "Full".
+   More information about PDB generation can be found [here](https://docs.microsoft.com/visualstudio/debugger/how-to-set-debug-and-release-configurations?view=vs-2019). If you are targeting the new runtime, please make sure that "Debugging Information" is set to "Portable", in order to properly generate Portable-PDB. If you are targeting the legacy runtime, you need to use "Full".
```
Comment (gewarren) [inline docs/cross-platform/using-visual-studio-tools-for-unity.md]: (Unrelated to your changes) - Can you remove `?view=vs-2019` from this link?

Comment (nschonni) [inline docs/cross-platform/using-visual-studio-tools-for-unity.md]:
```suggestion
   More information about PDB generation can be found [here](https://docs.microsoft.com/visualstudio/debugger/how-to-set-debug-and-release-configurations). If you are targeting the new runtime, please make sure that "Debugging Information" is set to "Portable", in order to properly generate Portable-PDB. If you are targeting the legacy runtime, you need to use "Full".
```

**Comment by PRMerger6** [comment]
@nschonni : Thanks for your contribution! The author, @mikeblome, has been notified to review your proposed change.

File: docs/cross-platform/using-visual-studio-tools-for-unity.md (hunk 1 above)
Comment (nschonni) [reply docs/cross-platform/using-visual-studio-tools-for-unity.md]: Done through a suggested change because I didn't want to pull down the branch 😄

**Comment by nschonni** [comment]
> Going forward, unless it's needed for a markdown lint rule to be turned on, we'd prefer it if you didn't make changes in that folder as we are not investing in that content.

Fair, I wish markdownlint had support for nested config so there could just be a separate config for folders to be ignored

File: docs/azure/vs-azure-tools-optimizing-azure-code-in-visual-studio.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ public async Task RunAsync()
 
     HttpClient client = new HttpClient();
 
-    Task<string> urlString = client.GetStringAsync("http://msdn.microsoft.com");
+    Task<string> urlString = client.GetStringAsync("https://msdn.microsoft.com");
```
[the remark this answers is not in the document]
Comment (nschonni) [reply docs/azure/vs-azure-tools-optimizing-azure-code-in-visual-studio.md]: I can undo it, but probably doesn't hurt to use HTTPS either

File: docs/debugger/graphics/graphics-event-list.md [hunk 3]
```diff
@@ -76,10 +76,10 @@ Use the Graphics Event List in Visual Studio Graphics Analyzer to explore the Di
...
 |---------------------| - | - |
-|Check user-defined event availability|[PIXGetStatus](https://msdn.microsoft.com/library/f7ebd985-fb5d-46d7-abec-099df4b9be0e)|[PIXGetStatus](http://msdn.microsoft.com/en-us/1046ac43-a0a3-42bf-bae8-14aa72fa7567)|
-|Begin an event group|[PIXBeginEvent](http://msdn.microsoft.com/en-us/5f51fff7-f313-4558-965b-2a443653cd7b)|[PIXBeginEvent](http://msdn.microsoft.com/en-us/4ddb3311-b9b5-449a-bbfb-7634e0d56e87)|
-|End   an event group|[PIXEndEvent](http://msdn.microsoft.com/en-us/fb526bf2-c17d-4a2a-8665-3b577a0f7fba)|[PIXEndEvent](http://msdn.microsoft.com/en-us/a3cd34a9-9dd9-40e1-ae86-0214b25ff185)|
-|Create an event marker|[PIXSetMarker](http://msdn.microsoft.com/en-us/0caf49ed-c99d-405e-89f4-0c887b8474ad)|[PIXSetMarker](http://msdn.microsoft.com/en-us/6610e5b9-a0c5-4236-b551-b6eb9fac64c1)|
+|Check user-defined event availability|[PIXGetStatus](https://msdn.microsoft.com/library/f7ebd985-fb5d-46d7-abec-099df4b9be0e)|[PIXGetStatus](https://msdn.microsoft.com/1046ac43-a0a3-42bf-bae8-14aa72fa7567)|
```
[the remark this answers is not in the document]
Comment (Mikejo5000) [reply docs/debugger/graphics/graphics-event-list.md]: @gewarren Thanks, I'll take a look at it.

**Pull request merged**